files/delete

```diff
@@ -13,7 +21,10 @@ if [ "$vm_uuid" ]; then
 fi
 
 if [ "$vdi_uuid" ]; then
+    # destroy volume
     xe vdi-destroy uuid="$vdi_uuid"
+    # remove symlink
+    rm -f "$dir_links/$vm_name_label"
```

Comment:
After removing the symlink, we require to issue a scan() to do the actual vdi_destroy.